package/gargoyle/files/www/templates/password_confirm_template

```diff
@@ -11,7 +11,7 @@
 
 				<div class="row form-group">
 					<span class="col-xs-12">
-						<input type="password" id="password" class="form-control"/>
+						<input type="password" id="password" class="form-control" autocomplete="new-password"/>
```

Comment:
This isn't correct.
Currently the only (?) use of this modal is to confirm that you wish to format an external USB drive. Therefore it requires your current password, not a suggested new one.

Since we may use this modal in different ways in the future, we may need to be careful here. But certainly at this point, it should be "current-password"

Reply:
Thanks for pointing it out. I will change it to "current-password".

Reply:
If the modal will be used differently in the future, it could conditionally include one of two more general templates, `current_password_template` or `new_password_template`, just for the input field or container.